articles/cosmos-db/mongodb-mongochef.md

```diff
@@ -17,6 +17,9 @@ To connect to a Cosmos account using Azure Cosmos DB's API for MongoDB, you must
 * Download and install [Studio 3T](https://studio3t.com/)
 * Have your Cosmos DB [connection string](connect-mongodb-account.md) information
 
+> [!NOTE]
+> There are known issues with the latest version of Robo 3T (v1.3) which are expected to be fixed soon.
```

Comment:
Please mention the version of Robo3T which works so that the customer can downgrade and self-mitigate

Reply:
+1 on Siddhesh's comment, rather than saying that there are known issues, it's best to mention the version that works. I would phrase the content as follows:

"Currently you can use Azure Cosmos DB's API for MongoDB with x version of Robo 3T, versions higher than this are not yet supported" 

Also, the doc you updated is about Studio 3T, isn't it different from Robo 3T? looks like we have another doc for Robo 3T and this is where the note can be added ?  https://docs.microsoft.com/en-us/azure/cosmos-db/mongodb-robomongo

Reply:
Makes sense - I updated the text to "Currently, Robo 3T v1.2 and lower are supported with Cosmos DB's API for MongoDB."